### ISSUE TITLE
Bug: playback pauses between files

### ISSUE THREAD

**Issue by Cptbluetooth** [TortugaPower/BookPlayer#833]
## Environment

 - OS Version: 16.0 (20A362)
 - App Version: 4.7.0

playback pauses between audiobook files. I’ve noticed this for the last 2 versions. 

## Steps to reproduce the behavior
1. Play a book with multiple files
2. Wait for one file to end and it will pause playback

**Comment by GianniCarlo** [comment]
Yeah this has happened to me as well, and it's in queue to be fixed for the next minor release 👌

**Comment by GianniCarlo** [comment]
Solved in v4.7.2 🙌
